src/control/AxesController.ts

```diff
@@ -183,6 +186,26 @@ class AxesController {
     return this;
   }
 
+  public addPreventClickHandler() {
+    const flicking = getFlickingAttached(this._flicking, "Control");
+    const axes = this._axes!;
+    const viewportEl = flicking.element;
+
+    axes.on(AXES.EVENT.HOLD, this._onAxesHold);
+    axes.on(AXES.EVENT.CHANGE, this._onAxesChange);
+    viewportEl.addEventListener("click", this._preventClickWhenDragged, true);
```

Comment:
Do you know how far to prevent the click event?

If a click event is prevented even if the drag pointer leaves the viewport, it seems to be necessary to add a click capturing event targeting the window.

Reply:
In what specific case do you mean?
Point of the click event preventing is for panel elements, so I think it's okay as they're all in the viewport elements.